### PR TITLE
GsaGH-402 Footfall COM result fix

### PR DESCRIPTION
**Pull request by kpne**

### CODE REVIEW

File: GsaGH/Helpers/GsaCOM/ComResultsHelper.cs [hunk 1]
```diff
@@ -55,12 +56,16 @@ private enum ElementDimension {
...
-        r.XyzResults.TryAdd(nodeId, xyzRes);
+        try {
+          var ff = new Ratio((double)gsa.Output_Extract(nodeId, 0), RatioUnit.DecimalFraction);
+          var res = new GsaResultQuantity() {
+            X = ff,
+          };
+          xyzRes.TryAdd(0, res);
+          r.XyzResults.TryAdd(nodeId, xyzRes);
+        } catch (System.ArgumentException) {
+          // skip
```
Comment (tlmnrnhrdt) [inline GsaGH/Helpers/GsaCOM/ComResultsHelper.cs:67]: when does it throw `ArgumentExceptions`? should we inform the user?

Comment (kpne) [reply GsaGH/Helpers/GsaCOM/ComResultsHelper.cs:67]: When it cannot find footfall result values for a particular node. This is an edge-case, where the user has included points in a 2D Member but those points lie withing void cutters or a cut out by section shapes:
image
The current method is returning max footfall result (transient or resonant) and if the node does not have a value we can simply skip it.

Comment (tlmnrnhrdt) [reply GsaGH/Helpers/GsaCOM/ComResultsHelper.cs:67]: Maybe worth putting a comment there ;)

**Review by tlmnrnhrdt — APPROVED** [review]

File: GsaGH/Helpers/GsaCOM/ComResultsHelper.cs (hunk 1 above)
Comment (kpne) [reply GsaGH/Helpers/GsaCOM/ComResultsHelper.cs:67]: We are hopefully getting rid of this class soon with [GSA-7392](https://ovearup.atlassian.net/browse/GSA-7392)

**Pull request merged**